styles/MyInvision/theme/mchat_custom.css

```diff
@@ -0,0 +1,12 @@
+.mchat-wrapper .row-item {
+    top: -10px;
+}
+.mchat-wrapper .header {
+    padding-left: 15px;
+    padding-top: 10px;
+}
+#mchat-panel {
+    border-bottom: 1px solid #ccc;
+    border-left: 1px solid #ccc;
+    border-right: 1px solid #ccc;

```

Comment:
Please use tabs for indentation.

Reply:
Also, in the style's `style.cfg` it says `name = myInvision`. Will it work with `MyInvision` (upper-case M) as folder name?

Reply:
I've tested it on my test board and worked fine. Fixed the spaces

Reply:
There are now 8 spaces and no tab. Maybe it's your editor that inserts spaces when you hit tab?